docker-images/pysyft-worker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-slim
+
+ENV WORKER_DIR /worker
+
+# Install PySyft
+RUN apt-get update && apt-get install -y gcc
+RUN pip install --no-cache-dir syft
+
+# Add worker code
+RUN mkdir $WORKER_DIR
+WORKDIR $WORKER_DIR
+COPY ./entrypoint.sh .
+RUN chmod +x ./entrypoint.sh
+COPY ./worker-server.py .
```

Comment:
In seems these 2 COPY's can be replaced with 1:
COPY ["./entrypoint.sh", "./worker-server.py", "./"]

Reply:
I was thinking that worker-server.py is more prone for updates and so ending up building only the last layer, what's your thoughts about that?

Reply:
This makes sense :) However, you'll want to disable caching anyway if you want `pip install syft` to pull the latest version.

Reply:
right, I will update that one.